### DISCUSSION TITLE
Dark Matter/Non Matter

### DISCUSSION THREAD

**Discussion by Anonymous69God** [r-spacex/SpaceX-API#1226] [General]
1         :
According to me "Dark matter" it is a Form of Non Matter which contains energy which if *Circuit is closed* then it contains far more energy which is not even energy
We can clarify it as a mixture of energy ;
2         :
With the help of 
E=MC²
We get to know that in this equation
E/1 = 0/1--eq¹ or E = 0/1--eq²
And accordingly to me it is eq² 
Because every thing have some kind of energy which binds it up.
3    :
So I am willing to make a theory Which States that
The Dark Matter or Non Matter is produced when :->
1) A Black Hole sucks the normal Matter in and due to high mass it(The black hole) contains it compareses the normal Matter to that state in which it's Molecular structure
4     :
It's Molecular structure Get's mutated and it stops showing normal property of matter
2)When the cosmos was building up (Hypothetically speaking) the energy which was left turned into Dark Matter that's why it's so powerful than normal Particle of Matter.

**Comment by FabianaCampanari** [comment]
Cool
